src/backend/api/Fusion.Resources.Api/Controllers/Requests/DepartmentRequestsController.cs

```diff
@@ -19,6 +21,7 @@ namespace Fusion.Resources.Api.Controllers.Requests
     [ApiController]
     public class DepartmentRequestsController : ResourceControllerBase
     {
+
```

Comment:
Missing [EmulatedUserSupport] here? Is it possible to put this attribute on the controller instead of each of the functions?

Reply:
No, i do not want to expose this to endpoints which get data or saves data, as it will basically look like a different user did the operation... So we need to tread somewhat carefully here.